charts/openstack-cloud-controller-manager/Chart.yaml

```diff
@@ -9,3 +9,7 @@ maintainers:
   - name: eumel8
     email: f.kloeker@telekom.de
     url: https://www.telekom.com
+dependencies:
+  - name: common
+    version: 2.14.1
+    repository: https://charts.bitnami.com/bitnami
```

Comment:
I think I'd rather prefer to just copy this single function here.

Reply:
Uff, I'm highly against copying code, that's the whole point of using a central, standard library for this; to remove code duplication.

Then you'd have to keep track of the upstream changes and copy them as well.

Reply:
Sure, but how "standard" is this standard library? I see it's used extensively in charts around GitHub, but it still seems to be from a third party?

Reply:
Maybe it's not a "standard" by definition, but as you said it's extremely widely used across helm charts and bitnami, even though they're a third party, has made themselves a recognizable helm chart writer and supporter known for their quality, at least in my knowledge circle.